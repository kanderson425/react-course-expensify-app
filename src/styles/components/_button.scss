.button {
    border-radius: 3px;
    border: none;
    padding: $s-size;
    font-weight: 300;
    font-size: $font-size-large;
    color: white;
    display: inline-block;
    background: $blue;
    text-decoration: none;
    line-height: 1;
}

.button--link {
    background: none;
}