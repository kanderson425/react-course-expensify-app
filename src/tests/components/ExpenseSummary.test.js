import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpensesSummary';

test('should ccorrectly render ExpenseSummaray with 1 expense', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={235}/> );
    expect(wrapper).toMatchSnapshot();
});

test('should ccorrectly render ExpenseSummaray with multiple expenses', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={23} expensesTotal={23512342134} />);
    expect(wrapper).toMatchSnapshot();
});