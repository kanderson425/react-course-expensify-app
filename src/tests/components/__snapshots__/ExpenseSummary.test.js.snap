// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should ccorrectly render ExpenseSummaray with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should ccorrectly render ExpenseSummaray with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $235,123,421.34
  </h1>
</div>
`;
